dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -921,6 +959,176 @@ protected void preValidateLaunchArguments(List<String> args) {
         }
     }
 
+
+    /**
+     * Creates a WES API object and sets the endpoint.
+     *
+     * @param wesUrl URL to WES endpoint
+     */
+    public WorkflowExecutionServiceApi getWorkflowExecutionServiceApi(String wesUrl) {
+        WorkflowExecutionServiceApi clientWorkflowExecutionServiceApi = new WorkflowExecutionServiceApi();
+        //ApiClient wesApiClient = clientWorkflowExecutionServiceApi.getApiClient();
+
+        // Done so we can override the Serialize method in ApiClient
+        // Since Swagger Codegen does not create correct code for the
+        // workflow attachement
+        ApiClientExtended wesApiClient = new ApiClientExtended();
+        clientWorkflowExecutionServiceApi.setApiClient(wesApiClient);
+
+        INIConfiguration config = Utilities.parseConfig(this.getConfigFile());
+        SubnodeConfiguration configSubNode = null;
+        try {
+            configSubNode = config.getSection("WES");
+        } catch (Exception e) {
+            //?????what should the return code be?????
```

Comment:
Use TODO or FIXME directives for this kind of thing.
http://wiki.c2.com/?FixmeComment

Reply:
Removed comment